packages/server/src/api/routes/tests/row.spec.ts

```diff
@@ -477,7 +483,7 @@ describe.each([
       expect(row.Story).toBeUndefined()
       expect(row).toEqual({
         ...defaultRowFields,
-        OrderID: 1111,
+        OrderID: createRowResponse.OrderID,
```

Comment:
SQL Server does not by default allow you to insert values into a primary key column.